cruise-control/src/yaml/endpoints/partitionLoad.yaml

```diff
@@ -0,0 +1,150 @@
+        schema:
+          type: integer
+          format: int64
+      - name: entries
+        in: query
+        description: The number of entries to show in the response
+        schema:
+          type: integer
+          default: 2147483647
+          format: int32
+      - name: topic
+        in: query
+        description: A regular expression used to filter the partition load returned based on topic
+        schema:
+          type: string
+          default: null
+      - name: partition
+        in: query
+        description: The partition or partition range to filter partition load returned
+        schema:
+          oneOf:
+            - type: integer
+              format: int32
+              example:
+                - value: 0
+                - summary: returns 0th partition
+            - type: string
+              pattern: '^[0-9]+-[0-9]+$'
+              example:
+              - value: '0-9'
+              - summary: 0 is the start index and 9 is the end index of the partition boundaries
+      - name: min_valid_partition_ratio
+        in: query
+        description: The minimum required ratio of partition load data completeness
+        schema:
+          type: number
+          default: null
+          format: double
+          minimum: 0.0
+          maximum: 1.0
+      - name: allow_capacity_estimation
+        in: query
+        description: Whether to allow capacity estimation when cruise-control is unable to obtain all per-broker capacity information
+        schema:
+          type: boolean
+          default: true
+      - name: max_load
+        in: query
+        description: If true, the maximum load is returned
+        schema:
+          type: boolean
+          default: false
+      - name: avg_load
+        in: query
+        description: If true, the average load is returned
+        schema:
+          type: boolean
+          default: false
+      - name: brokerid
+        in: query
+        description: Set of broker ids used to filter partition load returned
+        schema:
+          type: array
+          default: []
+          items:
+            type: integer
+            format: int32
+      - name: json
+        in: query
+        description: Whether to return in JSON format or not
+        schema:
+          type: boolean
+          default: false
+    responses:
+      '200':
+        description: |-
+          Successful partition load state response
+        content:
+          application/json:
+            schema:
+              $ref: '../responses/partitionLoadState.yaml#/PartitionLoadState'
+          text/plain:
+            schema:
+              type: string
+      '202':
+        description: |-
+          Partition load state fetch in progress
+        content:
+          application/json:
+            schema:
+              $ref: '../responses/progressResult.yaml#/ProgressResult'
+          text/plain:
+            schema:
+              type: string
+      '400':
```

Comment:
(Also apply to `rebalance.yaml`)the error response for `400`,`403` and `500` share the same format, can we just define once using the [default response syntax](https://swagger.io/docs/specification/describing-responses/)?

Reply:
Ah didn't realize that was a thing, that's much better